lib/tapioca/symbol_loader.rb

```diff
@@ -0,0 +1,81 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "json"
+require "tempfile"
+
+module Tapioca
+  module SymbolLoader
+    class << self
+      extend T::Sig
+      include SorbetHelper
+      include Reflection
+
+      sig { returns(T::Set[String]) }
+      def core_symbols
+        unless @core_symbols
+          output = symbol_table_json_from("-e ''", table_type: "symbol-table-full-json")
+          @core_symbols = T.let(SymbolTableParser.parse_json(output), T.nilable(T::Set[String]))
```

Comment:
The name `core_symbols` doesn't feel informative enough to me. Is there somewhere else that we use this terminology? If not maybe it's better to use a different name for this?

Reply:
I went with `payload_symbols` instead